scripts/build.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function init() {
+    RED='\033[0;31m'
+    BLUE='\033[0;34m'
+    BOLD='\033[1m'
+    NONE='\033[0m'
+
+    REPO_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}")/../" && pwd )"
+}
+
+function print_usage() {
+    echo -e "\n${RED}Usage${NONE}:
+    ${BOLD}$0${NONE} [OPTION]"
+
+    echo -e "\n${RED}Options${NONE}:
+    ${BLUE}test${NONE}: run all unit tests
+    ${BLUE}check_style${NONE}: run code style check
+    "
+}
+
+function abort(){
+    echo "Your change doesn't follow PaddlePaddle's code style." 1>&2
+    echo "Please use pre-commit to check what is wrong." 1>&2
+    exit 1
+}
+
+function check_style() {
+    trap 'abort' 0
+    set -e
+
+    export PATH=/usr/bin:$PATH
+    pre-commit install
+    clang-format --version
+
+    if ! pre-commit run -a ; then
+        git diff
+        exit 1
+    fi
+
+    trap : 0
+}
+
+function run_test() {
+    apt-get install -y cmake
+    mkdir -p ${REPO_ROOT}/build
+    cd ${REPO_ROOT}/build
+    cmake ..
```

Comment:
Do we need to specify `PADDLE_PYTHON_PATH` here, as required [here](https://github.com/PaddlePaddle/PARL/pull/10/commits/7f095164a48671efade57f7e52ca70ef9d598ebe#diff-af3b638bc2a3e6c650974192a53c7291L20)

Reply:
We will use a production docker image for testing, and don't need to specify python path.